packages/syft/src/syft/util/telemetry.py

```diff
@@ -85,4 +84,4 @@ def noop(func: T) -> T:
         return noop
 
 
-instrument = setup_tracer()
+instrument: Callable = setup_tracer()
```

Comment:
This one is tricky. `instrument` must have type `Callable[Concatenate[T, ..], T]`, not just `Callable`, otherwise we lose typing and auto-completion for any class that use the `@instrument` decorator. However, mypy does not seem to be able to infer the type for `instrument` if we just do `instrument = setup_tracer()` and if we set `instrument: Callable[Concatenate[T, ..], T] = setup_tracer()` it complains about wrong typing too. We'd have to modify the code a bit. I'll push a fix for this.